example/http/client/client.go

```diff
@@ -17,10 +17,11 @@ package main
 import (
 	"context"
 	"fmt"
-	"go.opentelemetry.io/plugin/httptrace"
 	"io/ioutil"
 	"net/http"
 
+	"go.opentelemetry.io/plugin/httptrace"
+
```

Comment:
nit: should this be moved next to "go.opentelemetry.io/api/key" ?

Reply:
Oh yeah, this change was made by the linter. Seems that a human is still needed to lint the linter's work. ;)